src/libraries/listener.library.ts

```diff
@@ -1,11 +1,29 @@
 import { BlackListedKeyboardShortcutChecker } from './black-listed-key-bindings.library';
 
-export interface IKeyboardShortcutListenerOptions {
+export interface IKeyboardShortcutListenerOptions { // defaults with ?
```

Comment:
we should have an @description block on this object with something like "represents an object that is used in the construction of a new listener"